default/stringtables/en.txt

```diff
@@ -6433,7 +6433,16 @@ A set of core supply starlanes for an empire (the set of least-jump starlanes co
 
 Supply lines can also be used to refill the [[metertype METER_FUEL]] supply of ships.
 
-Supply lines will be obstructed by armed enemy ships belonging to empires with which an empire is at [[encyclopedia WAR_TITLE]]. At the opposite end of the spectrum, empires in an [[encyclopedia ALLIANCE_TITLE]] will share supply lines and all their associated benefits.'''
+Supply lines will be obstructed by armed enemy ships belonging to empires with which an empire is at [[encyclopedia WAR_TITLE]]. At the opposite end of the spectrum, empires in an [[encyclopedia ALLIANCE_TITLE]] will share supply lines and all their associated benefits.
+
+If multiple empires compete for supply in a system, the empire with the highested propagated supply will win. In order to break ties, a bonus per empire is calculated and added to the propagated supply before comparison.
+• Has a colony in the system: +0.5
+• Has an outpost in the system: +0.3
+* Has supply sources in the system: +(0.001 * SUM(own supply in the system))
+* Has supply sources in the system: +(0.00001 * SUM(own max supply in the system))
+* Total supply: +(0.00000001 * SUM(own supply in the empire))
+* Has supplying source outside the system: +(0.0001 * MAX(1.0, MAX(distance to supplying system)))
```

Comment:
```suggestion
* Distance to the source of supply that reaches the system and that, before propagation, is the strongest. That is, a 4 strength source 4 jumps away is stronger than a 3 strength srouce 3 jumps away: +(0.0001 * distance to supplying system)
```

Reply:
i had a hard time analysing what empire_supply_it actually contains.

Is `empire_supply_it->second.second` the original (not propagated) supply strength? so a 4 strength system 2 jumps away is stronger than a 3 strength system 3 jumps away? in that case the distance does not matter.
 
Or does it contain the distance? in that case the 3 strength system 3 jumps would be stronger

Reply:
ah, just saw your explanations on the forum for the dominant supply source, thanks.